smtp_test.go

```diff
@@ -0,0 +1,16 @@
+package glutton 
+
+import (
+	"testing"
+)
+
```

Comment:
use gofmt. I recommend using an IDE like VSCode, Atom or GoLand

Reply:
Yeah I do use VSCode, planning to shift to golang soon